### PR TITLE
[PM-9842] Expired link UI

### PR DESCRIPTION
**Pull request by andrebispo5**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-9842

## 📔 Objective

Add necessary classes for the expired link UI.

## 📸 Screenshots

image

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: BitwardenShared/UI/Auth/AuthRoute.swift [hunk 1]
```diff
@@ -84,6 +88,10 @@ public enum AuthRoute: Equatable {
     ///
     case startRegistration
 
+    /// A route to start registration screen when coming from the expired link view.
```
Comment (fedemkr) [inline BitwardenShared/UI/Auth/AuthRoute.swift:91]: ⛏️ To keep the same wording use `screen`:
```suggestion
    /// A route to start registration screen when coming from the expired link screen.
```

**Comment by KatherineInCode** [comment]
⛏️  Can the name of the PR (/message on the eventual merge commit) be updated with `[PM-9842]`?

**Comment by andrebispo5** [comment]
> ⛏️ Can the name of the PR (/message on the eventual merge commit) be updated with `[PM-9842]`?

For sure, totally miss that! Thank you :)

File: BitwardenShared/UI/Auth/AuthRoute.swift (hunk 1 above)
Comment (fedemkr) [reply BitwardenShared/UI/Auth/AuthRoute.swift:91]: Yes, if you can and it's not too much work given that's outside of this task scope. Btw, I don't mind choosing either `view` or `screen` as long as we always use the same one.

**Review by fedemkr — APPROVED** [review]

**Pull request merged**